.changeset/forty-kings-tan.md

```diff
@@ -0,0 +1,7 @@
+---
+"@vue-storefront/sdk": patch
+---
+
+[CHANGED] Headers with same name will now have their values merged using the ";' delimiter, rather than ",". This is seen when middleware and defaultRequestConfig set the same header - e.g. `Cookie`. 
```

Comment:
```suggestion
[FIXED] Headers with same name will now have their values merged using the ";' delimiter, rather than ",". This is seen when middleware and defaultRequestConfig set the same header - e.g. `Cookie`. 
```